packages/devtools_app/lib/src/screens/logging/metadata.dart

```diff
@@ -201,6 +201,8 @@ class KindMetaDataChip extends MetadataChip {
     String? kindIconAsset;
     if (kind == 'stdout' || kind == 'stderr') {
       kindIcon = Icons.terminal_rounded;
+    } else if (kind == 'TimerSignificantlyOverdue') {
```

Comment:
I realized that I forgot to update the `EventKind` enum in `package:vm_service` to include `kTimerSignificantlyOverdue`. I'll do that now and update this line.

Reply:
Sounds good - I will wait for that change to be applied.